### PR TITLE
enable_word_time_offsets=False to a few examples, and added enable_wor…

### PR DESCRIPTION
**Pull request by dlaqab**
Added enable_word_time_offsets=True to local file version of transcribe_async
Added enable_word_time_offsets=False to other recognize samples (except streaming)

### CODE REVIEW

**Comment by dlaqab** [comment]
I am covered by employment agreement

**Comment by jonparrott** [comment]
@dlaqab you need to register your github username at go/github, and make sure your @google.com address is associated with your github profile.

File: speech/cloud-client/quickstart.py [hunk 1]
```diff
@@ -46,7 +46,8 @@ def run_quickstart():
     config = types.RecognitionConfig(
         encoding=enums.RecognitionConfig.AudioEncoding.LINEAR16,
         sample_rate_hertz=16000,
-        language_code='en-US')
+        language_code='en-US',
+        enable_word_time_offsets=False)
```
Comment (jonparrott) [inline speech/cloud-client/quickstart.py]: If this defaults to false, then we should just leave it out.

Comment (dizcology) [inline speech/cloud-client/quickstart.py]: Agreeing with @jonparrott on this:  This is definitely something we want to make clear in the API/client library reference, but adding this in the sample codes would make the API seem harder to use than it actually is (note that the other parameters `encoding`, `langauge_code`, and `sample_rate_hertz` are all required, but `enable_word_time_offsets` is optional and defaults to false).

What are some possible use case scenarios where not showing `enable_word_time_offsets=False` could cause friction for the users?

**Comment by dlaqab** [comment]
@jonparrott thanks for the pointer - Done

**Comment by remi** [comment]
The enable_word_time_offsets=False parameter in these samples is extraneous to the task these samples are intended to demonstrate. Closing.

**Comment by remi** [comment]
Note: samples which demonstrate Enable Word Time Offsets were added to java-docs-samples in #1050

**Pull request closed without merging**